### PR TITLE
Add TT1 blocks

### LINKED ISSUE #17: Add TT1 blocks
I propose adding TT1 Blocks to this repository. I would like to see it updated.
https://wordpress.org/themes/tt1-blocks/

### PR DESCRIPTION
**Pull request by MaggieCabrera**
This PR adds TT1 blocks to this repo. We should continue the work started in https://github.com/WordPress/theme-experiments/pull/307 on this repo now and anything else that needs updating since the last time this theme had any work done.

/cc @carolinan 

Closes https://github.com/WordPress/community-themes/issues/17

### CODE REVIEW

File: tt1-blocks/index.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?php
```
Comment (MaggieCabrera) [inline tt1-blocks/index.php:1]: I think this file can go

File: tt1-blocks/functions.php [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+ * @return void
+ */
+function tt1_blocks_block_editor_script() {
+
+	wp_enqueue_script( 'tt1-blocks-unregister-block-style', get_theme_file_uri( '/assets/js/unregister-block-style.js' ), array( 'wp-blocks', 'wp-dom' ), wp_get_theme()->get( 'Version' ), true );
+}
+add_action( 'enqueue_block_editor_assets', 'tt1_blocks_block_editor_script' );
+
+// Block Styles.
+require get_template_directory() . '/inc/block-styles.php';
```
Comment (MaggieCabrera) [inline tt1-blocks/functions.php:91]: I think everything in this file except the style shared/blocks stylesheets and the block styles stuff can go, don't you think @carolinan ? 

We can also discuss if those stylesheets can be consolidated into one if they end up being necessary at all

**Comment by MaggieCabrera** [comment]
I think let's merge this one and continue working on separate PRs so the diffs are more meaningful to review, I think this still has a lot of work and there's a lot we can remove.

**Comment by richtabor** [comment]
I don't think this theme should be part of the repo. It's quite dated/uses a lot of CSS, functions, and isn't a great example of what a top-notch block theme is (today). 

There has been a lot of developments around theming since this was originally released.

**Comment by carolinan** [comment]
It wont be dated once we are ready to release the update.  🙂

**Comment by carolinan** [comment]
To extend on my comment, yes it is dated right now, I am suggesting breaking backwards compatibility and updating the minimum required version to 6.2 and modernizing it.
Updating the minimum required version will have a minimal impact because: `Active Installations: 100+`

**Comment by MaggieCabrera** [comment]
The reason why I think it's a good idea to include it in this repo is because TT1 blocks is already a community theme. It's already released under the WP username, and it's not being maintained. This way at the very least we ensure that the code is updated should people want to reference it at any point.

File: tt1-blocks/index.php (hunk 1 above)
Comment (MaggieCabrera) [reply tt1-blocks/index.php:1]: we can do it on a follow up where we also update the min required version of WordPress for the theme

**Comment by scruffian** [comment]
I think we should remove the functions.php and all the CSS files.

**Comment by MaggieCabrera** [comment]
> I think we should remove the functions.php and all the CSS files.

+1, we should lower the burden of maintenance as much as we can

**Comment by MaggieCabrera** [comment]
I opened issues for the removal of the code and other updates, if someone can ✅ this one, we can get the theme in the repo and start working on it?

File: tt1-blocks/functions.php (hunk 2 above)
Comment (carolinan) [reply tt1-blocks/functions.php:91]: I agree, and I think with more testing, many if not all the block styles should be removed too.

**Review by carolinan — APPROVED** [review]

**Comment by MaggieCabrera** [comment]
Thank you @carolinan let's make this one pretty

**Pull request merged**